templates/events/frontend/event-single/event.html

```diff
@@ -101,17 +101,33 @@ <h1 class="col-md-12">
         {# Start mobile-only sections ....... #}
 
         <div class="event-location event-location-mobile visible-xs">
-            <h2 class="h4 alt heading-location">Location:</h2>
+            <h2 class="h4 alt heading-location">Location(s):</h2>
```

Comment:
```suggestion
            <h2 class="h4 alt heading-location">Location{% if events_instance_location.pk and event_instance.virtual_url %}s{% endif %}:</h2>
```

We can make this label dynamic with this little snippet (not tested, but it _should_ work). I'm not sure if it's worth the trouble or not, but just an idea.

Reply:
Love that idea! thanks!